changelogs/fragments/57674-postgres_modules_use_exec_sql_instead_of_methods.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - postgresql modules - use module_utils.postgres.exec_sql function instead of __exec_sql method (https://github.com/ansible/ansible/pull/57674)
```

Comment:
```suggestion
  - postgresql modules - use ``module_utils.postgres.exec_sql`` function instead of ``__exec_sql`` method (https://github.com/ansible/ansible/pull/57674)
```